chart/templates/iq-server-jobs.yaml

```diff
@@ -37,6 +52,10 @@ spec:
               name: {{ .Release.Name }}-iq-server-secrets-volume
             {{- end }}
           env:
+            {{- range .Values.iq_server.env }}
+            - name: {{ .name }}
+              value: {{ .value | quote }}
```

Comment:
You should support valueFrom here as well, so secrets can be used

Reply:
Done.